### ISSUE TITLE
Muons: Capture Oscilloscope Pictures

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#6084]
As a muon scientist I want to be able to capture pictures of the oscilloscope screen when using it in RF experiments.  I want to be able to capture one-off images and automatically save a series of images at regular intervals.

#### Acceptance Criteria
1. Using the IBEX client, I can configure the IP address of my oscilloscope.
1. Using the IBEX client, I can set the frequency at which images are captured.
1. Using the IBEX client, I can set starting Sequence Number for labelling the captured images.
1. Using the IBEX client, I can set the name of the folder in which captured images will be saved.
1. Using the IBEX client, I can view the name of the most recently saved captured image.
1. Using the IBEX client, I can capture a single image of my oscilloscope screen (e.g. a button to capture a single image).
1. Using the IBEX client, I can capture a series of images of my oscilloscope screen (e.g. buttons to start and stop the automated capture of images ).

#### Notes
1. The purpose of this feature is to replicate the functionality of the `Muon Oscilloscope.vi` and `Scope Picture Logger.vi` VIs in `C:\LabVIEW Modules\Muon RF`.
   * The LabVIEW solution uses a VI, naturally, to implement the image capture feature.
   * In IBEX, it could be done with an OPI, speaking to the existing oscilloscope IOCs, but other solutions might be available.
1. The RF equipment moves between different muon instruments, so it is necessary to make the IP address of the oscilloscope configurable.

**Comment by John-Holt-Tessella** [comment]
Are the captured images scientific data and does it need to be archived with the current run?

**Comment by kjwoodsISIS** [comment]
No.  The scientist is free to choose which directory the images get stored in.  While the scientist could _choose_ to save the images with the run data, he/she could equally _choose_ to save them elsewhere.  The saving of oscilloscope image is not tied to the data archiving.
